console/src/components/Services/Actions/Common/components/HeaderConfEditor.js

```diff
@@ -32,15 +32,12 @@ const HandlerEditor = ({
         />
       </h2>
       <div className={`${styles.add_mar_bottom_mid}`}>
-        <label
-          className={`${styles.add_mar_right} ${styles.cursorPointer}`}
-          onClick={toggleForwardClientHeaders}
-        >
+        <label className={styles.add_mar_right}>
```

Comment:
Can you make the entire label (along with the checkbox) `cursor:pointer`?

Reply:
Sure @wawhal

Reply:
pushed